arc/src/zjs_i2c_arc.json

```diff
@@ -8,6 +8,9 @@
             "CONFIG_I2C=y"
         ]
     },
-    "src": ["arc_i2c.c", "../../src/zjs_ipm.c", "../../src/zjs_i2c_handler.c"],
+    "src": [
+        "src/arc_i2c.c",
+        "../../src/zjs_ipm.c",
+        "../../src/zjs_i2c_handler.c"],
```

Comment:
Are you sure this shouldn't be just one ../ ? Above it seemed like you're building from one dir higher now?

Reply:
actually, i don't get any compile errors if i remove them, seems they are not neccessary.